page.go

```diff
@@ -224,26 +224,23 @@ func (p *Page) UnmarshalJSON(b []byte) error {
 }
 
 func (p UpdatePageParams) Validate() error {
-	if p.DatabasePageProperties == nil && p.Title == nil {
-		return errors.New("either database page properties or title is required")
+	if p.DatabasePageProperties == nil {
+		return errors.New("database page properties is required")
 	}
 	return nil
 }
 
 func (p UpdatePageParams) MarshalJSON() ([]byte, error) {
```

Comment:
I think we can drop this method altogether; the default behaviour of marshalling `UpdatePageParams` by `encoding/json` _should_ just work now.